doc/manuals/install_guide.rst

```diff
@@ -16,16 +16,17 @@ Linux distributions may have different packages already installed, or
 may use a different package manager than apt, but even on Ubuntu this
 should help to provide a starting point.
 
-`sudo apt-get install git zlib1g-dev libcurl4-openssl-dev libexpat1-dev dh-autoreconf liblapack-dev libxt-dev`
+.. code::
 
-`sudo apt-get build-dep libboost-all-dev qt5-default`
+  sudo apt-get install git zlib1g-dev libcurl4-openssl-dev libexpat1-dev dh-autoreconf liblapack-dev libxt-dev
+  sudo apt-get build-dep libboost-all-dev qt5-default
```

Comment:
Does KWIVER use Qt?  I thought Qt was only needed for MAP-Tk and ViViA?

Reply:
I don't see any obvious evidence that it does. However, I also notice that `.travis.yml` refers to Qt (4!). Thus, I petition both of those for removal in a separate PR.